triage/nw_prj_classifier.py

```
@@ -0,0 +1,105 @@
+                date=admin.date,
+                resolved_hours=admin.hours,
+                admin_ref=admin,
+                roster_refs=rosters,
+                status="GREEN",
+                reason_code="RECONCILED"
+            )
+
+            # Apply Rich Guard
+            if admin.tech in self.protected_names:
+                # Rich afternoon clock-out check
+                has_afternoon_clockout = any(r.punch_out and "12:" not in r.punch_out and "1:" in r.punch_out or "2:" in r.punch_out or "3:" in r.punch_out for r in rosters)
```

Ruff (0.15.15)

[warning] Parenthesize `a and b` expressions when chaining `and` and `or` together, to make the precedence clear

Parenthesize the `and` subexpression

(RUF021)